frontend/server/src/Scoreboard.php

```diff
@@ -641,7 +640,7 @@ function (array $a, array $b): int {
     }
 
     /**
-     * @param array{username: string, name: string, country: null|string, is_invited: bool, place?: int, problems: array{alias: string, points: float, penalty: float, percent: float, runs: int}[], total: array{points: float, penalty: float}}[] $scoreboard
+     * @param array<int, array{country: null|string, is_invited: bool, name: string, problems: array<array-key, array{alias: string, penalty: float, percent: float, points: float, runs: int}>, total: array{penalty: float, points: float}, username: string}> $scoreboard
```

Comment:
nit:

```suggestion
     * @param list<array{country: null|string, is_invited: bool, name: string, problems: array<FIXME, array{alias: string, penalty: float, percent: float, points: float, runs: int}>, total: array{penalty: float, points: float}, username: string}> $scoreboard
```

qué tipo es ese `FIXME`? recuerda que `array-key` es subóptimo tenerlo porque es casi un `mixed`.

Reply:
Era un `array` normalito. Ya corregí los tipos.